spec/reek/smells/prima_donna_method_spec.rb

```diff
@@ -3,28 +3,19 @@
 require_relative 'smell_detector_shared'
 
 RSpec.describe Reek::Smells::PrimaDonnaMethod do
+  it 'reports the right values' do
+    src = 'class C; def m!; end; end'
+    expect(src).to reek_of :PrimaDonnaMethod,
+                           lines: [1],
+                           name: 'C',
```

Comment:
But this wouldn't make much sense AFAIS. [1] the message below even says "has prima donna method ...." and [2] the smell detector operates on class level, not on method level.
Sure we can fix [1] but [2] still suggests that the name should be 'C' here, not 'm'.